app/lib/team_migration/east_of_england.rb

```diff
@@ -71,6 +77,203 @@ def move_patients_in_generic_clinic
     old_session.destroy!
   end
 
+  def reassign_class_imports
+    ClassImport
+      .where(team: current_team)
+      .includes(location: :team)
+      .find_each do |class_import|
+        new_team = class_import.location.team
+        log(
+          "Reassigning class import #{class_import.id} to #{new_team.workgroup}"
+        )
+        class_import.update!(team: new_team)
+      end
+  end
+
+  def reassign_cohort_imports
+    CohortImport
+      .where(team: current_team)
+      .includes(patients: :teams)
+      .find_each do |cohort_import|
+        new_team =
+          cohort_import.patients.flat_map(&:teams).find { it.id.in?(team_ids) }
+
+        if new_team.nil?
+          log("No team for cohort import #{cohort_import.csv_filename}")
+          cohort_import.destroy!
```

Comment:
These are cases where there was a validation error on the import, and therefore we have no means of determining which team they belong to.